doc/theory/sh_basis.rst

```diff
@@ -0,0 +1,84 @@
+.. _sh-basis:
+
+========================
+Spherical Harmonic basis
```

Comment:
The plural of "basis" is "bases". I would use that here because it refers to the fact that there is more than one.

Reply:
Thanks. Addressed in 2ea22b5. I assume the filename is fine, though. Let me know if changing it would also be more appropriate.

Reply:
So leaving as it is for now. If the filename is to be changed, another PR will follow-up. Thanks Ariel.